### ISSUE TITLE
Formatter breaks class with trailing comment

### ISSUE THREAD

**Issue by scottdweber** [pinterest/ktlint#728]
Given a class that is defined like this:

```kotlin
class CoolThing (
    val thing1: String,
    val thing2: String
) // this is a comment
{
    // ...
}
```

## Expected Behavior
When running the ktlint formatter, output should be:

```kotlin
class CoolThing (
    val thing1: String,
    val thing2: String
) {
    // this is a comment
    // ...
}
```

## Observed Behavior
Actual output (0.36.0) is a class that no longer compiles:

```kotlin
class CoolThing (
    val thing1: String,
    val thing2: String
) // this is a comment {
    // ...
}
```

## Your Environment
* Version of ktlint used: `0.36.0`
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): `org.jlleitschuh.gradle:ktlint-gradle:9.2.1`
* Version of Gradle used (if applicable): `5.6`
* Operating System and version: `macOS`

**Comment by shashachu** [comment]
@scottdweber could you do me a favor and just run `lint` (not format) with `--verbose` to see which rule it is that's making that change? (I'm unsure how to pass that flag with the plugin, but I assume it's supported).

**Comment by scottdweber** [comment]
Looks like it is `curly-spacing`

**Comment by romtsn** [comment]
Fixed by #730
